twoaxistracking/tests/test_placeholder.py

```diff
@@ -0,0 +1,6 @@
+import pytest  # noqa: F401
+import twoaxistracking  # noqa: F401
+
+
+def test_placeholder():
+    pass
```

Comment:
This is just a dummy placeholder so that `pytest` has something to find.  When real tests are added we can delete this file.

Reply:
Didn't we decide on having the tests outside of the package? :o

Reply:
Oh, I thoughtlessly copied pvlib about that.  Here's some discussion about each approach: https://docs.pytest.org/en/latest/explanation/goodpractices.html#choosing-a-test-layout-import-rules

I don't feel strongly one way or the other so I'll leave it up to you -- feel free to move the tests folder to the root if you like :)